cfy_manager/components/restservice/db.py

```diff
@@ -84,23 +84,31 @@ def prepare_db():
     script_path = join(SCRIPTS_PATH, 'create_default_db.sh')
     tmp_script_path = temp_copy(script_path)
     common.chmod('o+rx', tmp_script_path)
-
+    server_username = pg_config['server_username'].split('@')[0]
```

Comment:
OK, evidently I'm misunderstanding something here- can we have a comment somewhere explaining why we're adding the domain suffix when from this it looks like it should work without?

Reply:
When we try to connect to Azure DBaaS with psql or any other method (SQLAlchemy) we need to supply the user as - `<username>@<azure_domain>`
But, in the PG DB itself, the user login name is saved as `<username>` only.
So when I want to `CREATE USER...` or `GRANT <username>...` etc... I need to give it the <username> only.
That's why we have the `@` mishmash...

Reply:
Urgh, that's ugly but makes sense.

Reply:
Aside from suggesting adding a comment there to explain why we're doing it then, I'll mark this as resolved.